chain/epoch_manager/src/test_utils.rs

```diff
@@ -74,23 +75,25 @@ pub fn epoch_info(
 }
 
 pub fn epoch_config(
-    epoch_length: BlockIndex,
-    num_shards: ShardId,
-    num_block_producers: ValidatorId,
-    num_hidden_validators: ValidatorId,
+    epoch_length: HeightDelta,
+    num_shards: NumShards,
+    num_block_producer_seats: NumSeats,
+    num_fishermen: NumFishermen,
     block_producer_kickout_threshold: u8,
     chunk_producer_kickout_threshold: u8,
     fishermen_threshold: Balance,
 ) -> EpochConfig {
     EpochConfig {
         epoch_length,
         num_shards,
-        num_block_producers,
-        block_producers_per_shard: get_num_block_producers_per_shard(
+        num_block_producer_seats,
+        num_block_producer_seats_per_shard: get_num_seats_per_shard(
```

Comment:
`num_block_producer_seats`, it returns speciflcally block producer seats, removing the words "block producer" from the name makes it unclear

Reply:
This functions splits Seats between Shards and doesn't know whose Seats are received in arguments.
It means we can reuse the function for Seats of any kind, not block producer only.
Do we really need to keep `get_num_block_producers_per_shard`?